lib/Logger.js

```diff
@@ -49,6 +54,40 @@ class Logger {
         Config.update({ dump: levelName });
     }
 
+    getLogLevel() {
+        return Config.level;
+    }
+
+    getDumpLevel() {
+        return Config.dump;
+    }
+
+    /**
+     *  This function creates a new Logger, from another one,
+     *  changing its submodule field.
+     *
+     *  @param {Logger} baseLogger - Base Logger to create a new one.
+     *  @param {String} submodule - The name of the submodule.
+     *  @return {Logger} - The new logger with the new submodule.
+     */
+    static updateSubmodule(baseLogger, submodule) {
```

Comment:
Few comments here:
1. What is the point of updating the submodule for a "module logger" ?
2. Why create a new logger in an "updateSubmodule" ? Maybe the function needs renaming
3. Should we not only use that approach for the requestLogger, as previously discussed ?